Back-end/index.js

```diff
@@ -225,3 +226,54 @@ app.post("/register", async (req, res) => {
     });
   }
 });
+
+app.get("/match", async (req, res) => {
+  let users = []; // Hold users when filtering by states
+  let matches = []; // Hold users when filtering by age
+  let results; // Similarity score
+  let map1 = new Map(); // Storing users with the similarity score as the key and the value using the user information
+  const { age, state, keyword } = req.body;
+  const data = await prisma.User.findMany({ // Gets users from the database
+    where: {
+      accountType: "Mentor", 
+    },
+  });
+  users.map((user) => {
+    if (user.state === state) {
+      users.push(user);
+    }
+  });
+  if (users.length === 0) {
+    // res.status(404).send(`No mentors exist in the state of ${state} yet! Will display mentors from all states`)
```

Comment:
Remove commented code.

Reply:
Comments are all gone in the main branch